packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -79,14 +79,16 @@ export class VaLink {
 
   /** @ts-ignore */
   private handleClick = (e: MouseEvent): void => {
-    if (this.disableAnalytics) return;
-    this.componentLibraryAnalytics.emit({
-      componentName: 'va-link',
-      action: 'click',
-      details: {
-        // TODO: add analytics event details
-      },
-    });
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'va-link',
+        action: 'click',
+        details: {
+          'link-click-label': this.text,
```

Comment:
Same thing here regarding the `link-click-label` naming.

Reply:
I've updated this to just `label` because of the way the vets-website script is transforming it for the datalayer.